### PR TITLE
{bio}[GCCcore/10.3.0] MotionCor2 v1.4.4

### PR DESCRIPTION
**Pull request by stefan-wolfsheimer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by stefan-wolfsheimer** [comment]
see also https://github.com/easybuilders/easybuild-easyblocks/pull/2541

File: easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+# CUDA is a build dependency to make sure it gets installed.
+# It's actually a runtime dependency, but that's handled in the wrapper to
+# make sure it doesn't conflict with whatever toolchain happens to be loaded.
+# Change CUDA version to match the latest one used in this version
+# of MotionCor2
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('patchelf', '0.12'),
+    ('CUDAcore', '11.3.1', '', True),
```
Comment (casparvl) [inline easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb:39]: Please change `CUDAcore` to `CUDA`. `CUDAcore` is no longer used https://github.com/easybuilders/easybuild-easyconfigs/commit/09ea3ce3ebb30d00cfc741b284c8659d8da3ca8f

File: easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# of MotionCor2
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('patchelf', '0.12'),
+    ('CUDA', '11.3.1', '', True),
+]
+
+skip_rpath_check = True
+
+postinstallcmds = ['patchelf --force-rpath --set-rpath $EBROOTCUDACORE/lib64:$EBROOTLIBTIFF/lib %(installdir)s/bin/MotionCor2_1.4.4_Cuda113']
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb:44]: This should probably only be done if the rpath easyconfig config option is set to true

File: easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('XZ', '5.2.5'),
+    ('LibTIFF', '4.2.0'),
+    ('jbigkit', '2.1'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.3.0-CUDA-11.3.1.eb:51]: And only LibTIFF should be needed here as mentioned in #14074

**Review by akesandgren — DISMISSED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3126.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/81f1546af31c4da4f074aaeb79ee5712 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3126.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/cfa8a3203dd29856a5987a5d5b45fd1c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @stefan-wolfsheimer!

**Pull request merged**